Functions/xASL_io_ReadDataPar.m

```diff
@@ -192,3 +203,48 @@
 end
 
 end
+
+%%xASL_io_ReadDataPar_ConvertCellArrays finds all the cell arrays entered as strings and fixes them
+function StructOut = xASL_io_ReadDataPar_ConvertCellArrays(StructIn)
+
+% Get fields
+listFields = fields(StructIn);
+
+% Iterate over fields
+for iField=1:length(listFields)
+	
+	% Finds the incorrect fields
+	if isstruct(StructIn.(listFields{iField}))
+		% Calls recursively the function for structs
+		StructOut.(listFields{iField}) = xASL_io_ReadDataPar_ConvertCellArrays(StructIn.(listFields{iField}));
+		
+	elseif ischar(StructIn.(listFields{iField})) && ~isempty(StructIn.(listFields{iField})) && ...
+			StructIn.(listFields{iField})(1) == '{' && StructIn.(listFields{iField})(end) == '}'
+		% For chars entered in {} brackets, brakes the entries in '' to individuals arrays with a cell array
+		iFirst = 2;
+		iApostrophe = strfind(StructIn.(listFields{iField}),39);
```

Comment:
Yes `'` would have been better, but `'\''` doesn't work and I couldn't find a better way to escape it. But I will add a commentary.

Reply:
OK, looks like `''''` does the job as well as `39`, but as long as it is understood in a comment that `39` means `'`

(I believe in Matlab a `'` is escaped by `''` rather than `\'`